client/src/components/Donation/DonationOptionsText.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DonationOptionsText = ({ isAlert }) => {
+  const Title =
+    'Want to make a bigger one-time donation, mail us a check, or give in other ways?';
+
+  return (
+    <>
+      {!isAlert && (
+        <h4>
+          <b>{Title}</b>
+        </h4>
+      )}
+      <p className='donation-options'>
+        {Title} Here are many{' '}
```

Comment:
The duplication of `Title` here looks quite strange. 
![image](https://user-images.githubusercontent.com/15801806/102770123-e95e9a00-4383-11eb-9a36-f02bbaf3d070.png)

Shouldn't it show either the h4 or the p?

Reply:
Thanks for catching that.